.github/workflows/start-ci.yml

```
@@ -0,0 +1,78 @@
+name: Label to staging
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    if: github.event.label.name == 'start-ci'
```

It will skip the whole job when the label is different than `start-ci` that way :)

---

Thanks! I'll change that over and remove the other checks. It wouldn't work for the other comment based job I copied this from